prow/plugins/trigger/pull-request.go

```diff
@@ -106,6 +106,45 @@ func handlePR(c Client, trigger plugins.Trigger, pr github.PullRequestEvent) err
 				return buildAll(c, &pr.PullRequest, pr.GUID, trigger.ElideSkippedContexts)
 			}
 		}
+		if pr.Label.Name == labels.OkToTest {
```

Comment:
Should't all of this go into the `github.PullRequestActionLabeled` above?

Edit: sorry, seems like I confused the indentation here